src/core/dfn-panel.js

```diff
@@ -60,12 +60,34 @@ function createPanel(dfn) {
 function dfnExportedMarker(dfn) {
   if (!dfn.matches("dfn[data-export]")) return null;
   return html`<span
-    class="dfn-exported"
+    class="marker dfn-exported"
     title="Definition can be referenced by other specifications"
     >exported</span
   >`;
 }
 
+/**
+ * @param {HTMLElement} dfn
+ * @param {NodeListOf<HTMLAnchorElement>} links
+ */
+function idlMarker(dfn, links) {
+  if (!dfn.hasAttribute("data-idl")) return null;
+
+  for (const anchor of links) {
+    const parentIdlBlock = anchor.closest("pre.idl");
```

Comment:
I'm not sure exactly what `links` are, but this doesn't seem quite right; at least applying this to the WebRTC spec doesn't yield the right result for the `RTCRtpSender` interface for instance.

Reply:
Maybe the `respec-pr` demo didn't replace the ReSpec script in WebRTC spec. Can you try replacing the ReSpec script with https://deploy-preview-3157--respec-pr.netlify.app/respec-w3c.js locally?

`links` here are anchor elements that reference this dfn BTW.

Reply:
> doesn't yield the right result
more specifically it links to the `RTCPeerConnection` IDL declaration rather than `RTCRtpSender`

Reply:
> Maybe the `respec-pr` demo didn't replace the ReSpec script in WebRTC spec. Can you try replacing the ReSpec script with https://deploy-preview-3157--respec-pr.netlify.app/respec-w3c.js locally?

For clarification, I'm using a local build of the PR and integrating it locally in the WebRTC spec - it's not a deployment problem as far as I can tell.

> links here are anchor elements that reference this dfn BTW.

Thanks - the problem is that any IDL fragment that uses the said interface as a type will be in that list of links and will match `closest("pre.idl")`; there is no reason that the first fragment will be the right one.

Reply:
@dontcallmedom can you check again?

Reply:
working great, now, thanks!